### DISCUSSION TITLE
[Bug]: Failed to send message. HTTP 429 - {"error": "slow down"}

### DISCUSSION THREAD

**Discussion by Airkek** [danny-avila/LibreChat#1156] [General]
### Contact Details

ke1jia@vk.com

### What happened?

When i make request with curl all works fine:
![image](https://github.com/danny-avila/LibreChat/assets/45855057/8d45092a-5b66-40f8-9bbb-d16e1ae34b86)

But when i make request with LibreChat (same token) i get 429 error:
![image](https://github.com/danny-avila/LibreChat/assets/45855057/ed438546-c920-4c0d-8993-f3e1d16f791e)

Proxy takes no effect. No matter which model I use.

Fun fact: if I enable at least one plugin all works fine:
![image](https://github.com/danny-avila/LibreChat/assets/45855057/55c1c5ff-ab35-4c8e-97df-881b5a4517ae)

### Steps to Reproduce

1. Contact me for my openai token ( t.me/keijia )
2. Make request

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Airkek** [comment]
Also, all worked fine yesterday

**Comment by berry-13** [comment]
It seems to be a problem with the rate limit of OpenAI. It works for me without any problems right now

You can find more informationi [here](https://community.openai.com/t/getting-hit-with-429-slow-down-error/482704/3)

**Reply by Airkek** [reply]
So why it's working with curl? 
No problems when using my key with same IP anywhere not in LibreChat (even in LibreChat's plugins all looks just fine)

**Reply by danny-avila** [reply]
i think it was curious that you said curl works fine, plugins as well. It's likely they are rate limiting specific payloads that might be the default pattern for the OpenAI endpoint due to the widespread DDoS: https://github.com/danny-avila/LibreChat/discussions/1154

I'm looking into it right now

**Reply by danny-avila** [reply]
I'm working on a fix #1157
